internal/service/networkmanager/vpn_attachment.go

```diff
@@ -0,0 +1,323 @@
+package networkmanager
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/networkmanager"
+	"github.com/hashicorp/aws-sdk-go-base/v2/awsv1shim/v2/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+
+	// "github.com/hashicorp/terraform-provider-aws/internal/flex"
+	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
+	"github.com/hashicorp/terraform-provider-aws/internal/tfresource"
+	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func ResourceVpnAttachment() *schema.Resource {
+	return &schema.Resource{
+		CreateWithoutTimeout: resourceVpnAttachmentCreate,
+		ReadWithoutTimeout:   resourceVpnAttachmentRead,
+		UpdateWithoutTimeout: resourceVpnAttachmentUpdate,
+		DeleteWithoutTimeout: resourceVpnAttachmentDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		CustomizeDiff: verify.SetTagsDiff,
+
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
Given that network manager can need some time to create resources, could we make those timeout customisable?